src/site/navigation/facility_no_drupal_page_sidebar_nav.drupal.liquid

```diff
@@ -6,113 +6,108 @@
     For an example of how to make a string look like this, look at in `src/site/layouts/person_profile.drupal.liquid` where sidebarData is being assigned.
 {% endcomment %}
 
-{% if buildtype != 'vagovprod' %}
-    <script nonce="**CSP_NONCE**" type="text/javascript">
-      window.sideNav = {{ sidebarData | json }};
-    </script>
```

Comment:
All I really did here is remove the `if buildtype != 'vagovprod'` block, and auto-format the template.